source/views/calendar/calendar-util.js

```diff
@@ -12,7 +12,8 @@ export function shareEvent(event: EventType): Promise<any> {
 	const title = event.title
 	const times = getTimes(event)
 	const location = event.location
-	const message = `${title}\n\n${times}\n\n${location}`
+	const description = event.description
+	const message = `${title}\n\n${times}\n\n${location}\n\n${description}`
```

Comment:
We should trim this string, so that we don't wind up copying 6 blank lines on an event with no location nor description.